tests/examples_arguments_syntax/line_chart_with_cumsum_faceted.py

```diff
@@ -0,0 +1,36 @@
+"""
+Faceted Line Chart with Cumulative Sum
+------------------------------
+This chart creates multiple line subcharts from the cumulative sum of a field, one for each category.
+"""
+# category: line charts
+import altair as alt
+from vega_datasets import data
+
+source = data.disasters()
+columns_sorted = ['Drought', 'Epidemic', 'Earthquake', 'Flood']
+
+alt.Chart(source).mark_line(
+    interpolate='basis' # Use linear interpolation with B-spline method
```

Comment:
I'm unsure why we would use b spline interpolation here. Unless we can add an explanation, I say let's remove it from both specs.

Reply:
- Removed line interpolation due to lack of clear justification